content/rs/clusters/maintenance-mode.md

```diff
@@ -156,7 +160,7 @@ After turning on maintenance mode for node 2, Redis Enterprise saves a snapshot
 
 Now node 2 has `0/0` shards because shards cannot migrate to it while it is in maintenance mode.
 
-## Toggle maintenance mode via API
+## Set maintenance mode via API
 
 You can also turn maintenance mode on or off via [REST API requests]({{<relref "/rs/references/rest-api">}}) to [<nobr>POST `/nodes/{node_uid}/actions/{action}`</nobr>]({{<relref "/rs/references/rest-api/requests/nodes/actions#post-node-action">}}).
 
```

Comment:
Another addition following the conversation in https://redislabs.atlassian.net/browse/RED-68917:

"Maintenance mode can be turned off without restoring the snapshot using the CLI commands:
rladmin tune node 1 max_listeners 100
rladmin tune node 1 quorum_only disabled"